### ISSUE TITLE
Incorrect spectral parameters for HESS J1708-410 from 2008A&A...477..353A ?

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-cat#115]
@kosack - I think I might have found a bug in the published spectral parameters from HESS J1708-410 from 2008A&A...477..353A. (Or I'm confused / making a mistake).

In gamma-cat, the spectral parameters and SED are here:
* https://github.com/gammapy/gamma-cat/blob/master/input/data/2008/2008A%2526A...477..353A/tev-000093.yaml
* https://github.com/gammapy/gamma-cat/blob/master/input/data/2008/2008A%2526A...477..353A/tev-000093-sed.ecsv

The spectral parameters are (we do data entry with flux norm in units of `1e-12 cm-2 s-1 TeV-1`):
```
spec:
  type: pl
  norm: {val: 2.7, err: 0.3, err_sys: 0.2}
  index: {val: 2.46, err: 0.08, err_sys: 0.2}
  ref: 1
  theta: 0.3d
  erange: {min: 0.50 , max: 60}
```

As you can see in this check plot, the spectral model doesn't match the points by a factor of ~ 3:
https://github.com/cdeil/gamma-cat-status/blob/master/sed_png/source_000093.png

As far as I can see, our data entry in gamma-cat does correctly reflect the parameters given in the paper:
https://www.aanda.org/articles/aa/full/2008/49/aa8516-07/aa8516-07.html

Looking at the paper plot suggests that it's the spectral model norm that is incorrect:
https://www.aanda.org/articles/aa/full/2008/49/aa8516-07/aa8516-07.html
By eye I see `e2dnde ~ 1e-11 erg cm-2 s-1` at `e = 1 TeV`, which is inconsistent with the published norm parameter of `dnde = 2.46e-12 cm-2 s-1 TeV-1`:
```
In [1]: import astropy.units as u

In [2]: dnde = 2.46e-12 * u.Unit('cm-2 s-1 TeV-1')

In [3]: energy = 1 * u.TeV

In [4]: (energy ** 2 * dnde).to('erg cm-2 s-1')
Out[4]: <Quantity 4.3258767255e-12 erg / (cm2 s)>
```

**Comment by cdeil** [comment]
Corresponding issue in the HESS-internal HGPS tracker: https://bitbucket.org/hess_software/hgps_paper/issues/114

**Comment by kosack** [comment]
It is possible it's a typo in the paper! I'll see if i can track down where the number came from.

**Comment by kosack** [comment]
I can only find an old result file where the fit was slighly different that what is in the paper . In that version, the norm is 5.6e-12  TeV/cm^2/s (closer to what the points show). Why this is different than what is in the paper is a mystery... It's possible it was just a typo or transcription error, since the points look similar. Finding the old *wobble_chain* fit parameters seems to be difficult, however since they are not in the logs.

![export_1708](https://cloud.githubusercontent.com/assets/11677812/26210494/05a26826-3bf0-11e7-9695-a164bb1a983f.png)

**Comment by cdeil** [comment]
@kosack - Thanks!

I've updated the power-law norm and index to what is shown in the legend of the plot you pasted in the last comment in https://github.com/gammapy/gamma-cat/commit/bb974b8840072346f06fc6b9f691538de0191238

As can be seen in our gamma-cat check plot (updated in https://github.com/cdeil/gamma-cat-status/commit/21f21a867e3a0781d90568b813f995723d1e4a52), now the model and the points match. I also verified by eye comparing to the plot in the paper: https://www.aanda.org/articles/aa/full/2008/49/aa8516-07/aa8516-07.html
It's exactly the same, the new values you gave match the plot in the paper.

@kosack - I think you (or the HESS GWG conveners, cc @parsonsrd, ) should publish an erratum.

I just checked one other source from that paper, not all of them. For HESS J1427-608 the model and points look fine (match each other and the paper plot): https://github.com/cdeil/gamma-cat-status/blob/master/sed_png/source_000068.png
